tests/run-tests.sh

```diff
@@ -177,28 +177,54 @@ run_test ()
     )
 
     STATUS=$?
-    case ${STATUS} in
-    # XXX Should this be abstracted out in solo5-run-virtio.sh?
-    0|2|83) 
-        LOGS=$(find ${TMPDIR} -type f -name ${NAME}.log.\*)
+    LOGS=$(find ${TMPDIR} -type f -name ${NAME}.log.\*)
 
-        STATUS=99
-        if [ -z "${WANT_ABORT}" ]; then
-            [ -n "${LOGS}" ] && grep -q SUCCESS ${LOGS} && STATUS=0
-        else
-            [ -n "${LOGS}" ] && grep -q ABORT ${LOGS} && STATUS=0
-        fi
-        if [ ${STATUS} -eq "0" ]; then
-            if [ -n "${WANT_QUIET}" -a -n "${LOGS}" ]; then
-                # XXX This is all just horribly fragile.
-                # If "quiet mode" was requested, there should not be any output
-                # on the console with "Solo5:" in it unless there was an error,
-                # with the exception of "Solo5: Halted" on non-QEMU hypervisors.
-                if sed -e '/^Solo5: Halted$/d' ${LOGS} | grep -q 'Solo5:'; then
-                    STATUS=99
+    if [ -z "${LOGS}" ]; then
+
+        wait
+        return 99
+    fi
+
+    case ${NAME} in
+    *.ukvm)
+        case ${STATUS} in
+        0|1) 
+            if   [ -z "${WANT_ABORT}" ] && [ "${STATUS}" -eq "0" ] \
```

Comment:
I don't understand this change. Are you looking at the logs _and_ the exit status? If so, why?

Reply:
Yes, it was checking for both the logs and the exit status. Just checking exit status now.

Reply:
Ok, I'll just trust you that it works, the test code is horrible anyways.